docs/Contribute.md

```diff
@@ -3,6 +3,13 @@ title: 'Contribute'
 ---
 # Contributing to the AlmaLinux project
 
+## Help with reporting bugs and making fixes
+
+Since AlmaLinux is a 1:1 binary compatible with RHEL, it should have the same bugs as the current release of RHEL. AlmaLinux, as a downstream of RHEL, recommends following an "upstream first" approach in order to help not just the AlmaLinux but the whole EL community:
```

Comment:
```suggestion
Since AlmaLinux aims to be as close to RHEL as possible, it should have nearly identical bugs as the current release of RHEL. AlmaLinux recommends following an "upstream first" approach in order to help not just the AlmaLinux but the whole EL community:
```